### DISCUSSION TITLE
Data Should Be Updated Regularly

### DISCUSSION THREAD

**Discussion by Bobablocks** [bumoonio/bumoon-assets#3] [Q&A] [answered]
This repository should be updated regularly to keep assets information verified and trustworthy, If those not the case on your plan, this should be moved to another owner

**Comment by Bobablocks** [comment]
> This repository should be updated regularly to keep assets information verified and trustworthy, If those not the case on your plan, this should be moved to another owner

or you can move it to our account

**Reply by bumoonio** [reply]
i got the idea, please contact me @email

**Comment by Bobablocks** [comment, marked as answer]
Ok, will fork it first before doing further shortly
